### PR TITLE
[Page Header] New page header for metrics

### PR DESCRIPTION
**Pull request by RyanL1997**
### Description
New page header for metrics

### Screenshots

Old Header page layout changes 
Screenshot 2024-08-14 at 2 43 28 PM
Screenshot 2024-08-14 at 2 43 37 PM

New Header changes 
Screenshot 2024-08-14 at 2 44 17 PM

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/metrics/top_menu/metrics_export.tsx [hunk 1]
```diff
@@ -62,20 +64,36 @@ const Savebutton = ({
...
+  }
+
+  return <>{children}</>;
+};
+
 const MetricsExportPopOver = () => {
   const availableObservabilityDashboards = useSelector(selectPanelList);
-  const [availableDashboards, setAvailableDashboards] = React.useState([]);
-  const [osdCoreDashboards, setOsdCoreDashboards] = React.useState([]);
+  const [availableDashboards, setAvailableDashboards] = useState([]);
```
Comment (RyanL1997) [inline public/components/metrics/top_menu/metrics_export.tsx:85]: unrelated, but unified the usage here.

File: public/components/metrics/top_menu/metrics_export.tsx [hunk 2]
```diff
@@ -50,32 +51,49 @@ const Savebutton = ({
 }) => {
   return (
     <EuiButton
+      size="s"
```
Comment (ps48) [inline public/components/metrics/top_menu/metrics_export.tsx:54]: I think based on mocks, we should add. `fill` prop here.

Comment (RyanL1997) [reply public/components/metrics/top_menu/metrics_export.tsx:54]: fixed

**Comment by TackAdam** [comment]
Everything looks good with the feature flag enabled, but have some concerns when it is disabled.

When disabling it do we want to keep the original formatting? It currently places the new save action between the span and super date picker.
Metrics_FeatureFlagDisabled

**Review by ps48 — APPROVED** [review]

**Pull request merged**